### DISCUSSION TITLE
fluent-cms

### DISCUSSION THREAD

**Discussion by agriffard** [OrchardCMS/OrchardCore#16695] [General]
I had a quick look at a new .NET cms project: https://github.com/fluent-cms/fluent-cms

It seems to have some interesting ideas to provide some headless features.
Inspired by Umbraco as I understand.
@hishamco also tested it, I think I saw him tweeting (xing? ;-) ) with the author.

There is a performance test to compare it with Strapi (popular headless cms):
https://github.com/fluent-cms/fluent-cms/blob/main/doc%2Fpeformance-tests%2Fperformance-test-fluent-cms-vs-strapi.md#5-data-collection
Strapi performances appear to be really low.

I was wondering how OC would compare with it to return blog posts via api.

**Comment by hishamco** [comment]
This is because it uses `SqlKata` & `Dapper`

**Comment by agriffard** [comment]
@sebastienros in his Performance session said that OC blog can handle 4200 rps.

**Comment by hishamco** [comment]
BTW I downloaded the repo and I will have a quick look and see how it is different from Strapi
